scenarios/05_The_Swamp_Things.cfg

```diff
@@ -40,14 +40,17 @@
         # Note: I had increased this income declaration to {9 6 3} before I got the gold carryover
         # from Battle of Barnon fixed; now that that works, though, it ought to be able to come
         # back down a bit:
-        {INCOME 7 5 3}
-        {GOLD 190 145 100} # keep this lower than 200 so that Reme's dialogue below always has a chance to fire (i.e., in the case in which the player is entering the scenario with no gold)
+        {INCOME4 7 5 3 1}
+        {GOLD4 170 140 110 80} # keep this lower than 200 so that Reme's dialogue below always has a chance to fire (i.e., in the case in which the player is entering the scenario with no gold)
         recruit=Akladian Wiseman,Akladian Clansman,Akladian Warrior
         [modifications]
             {TRAIT_LOYAL}
 #ifdef EASY
             {TRAIT_STRONG}
 #endif
+#ifdef NIGHTMARE
```

Comment:
Don't recommend debuffing the player's units.

Reply:
[086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).